snmp/tests/test_e2e_core.py

```diff
@@ -301,3 +301,44 @@ def test_e2e_meraki_cloud_controller(dd_agent_check):
 
     aggregator.assert_metric('snmp.sysUpTimeInstance', count=2, tags=common_tags)
     aggregator.assert_all_metrics_covered()
+
+
+def test_e2e_core_detect_metrics_using_apc_ups_metrics(dd_agent_check):
+    config = common.generate_container_instance_config([])
+    instance = config['instances'][0]
+    instance.update(
+        {
+            'snmp_version': 1,
+            'community_string': 'apc_ups_no_sysobjectid',
```

Comment:
why do you need a rec with no sysbojectid ?

Reply:
This is to make 100% sure that the code is not relying on SysObjectID (since it's removed) to know what metrics to use.